src/x3d/auto_generated_node_helpers/x3dnodes_text.inc

```diff
@@ -66,12 +66,16 @@ begin
   FdMaxExtent.Send(Value);
 end;
 
-procedure TTextNode.SetString(const Value: array of string);
+procedure TTextNode.SetString(const Value: array of String);
+var
```

Comment:
Note that you left modifications to auto-generated file here :)

Reply:
Yes, just noted this one too :) Fixed.